meshtastic/tests/test_mesh_interface.py

```
@@ -1393,3 +1399,1045 @@ def send_texts() -> None:
+                    "batteryLevel": 55,
+                    "voltage": 4.1,
+                    "channelUtilization": 1.5,
+                    "airUtilTx": 0.5,
+                    "uptimeSeconds": 123,
+                }
+            }
+        }
+        monkeypatch.setattr(
+            iface,
+            "sendData",
+            lambda payload, *args, **kwargs: telemetry_calls.append((payload, kwargs)),
```

Ruff (0.15.2)

[warning] Unused lambda argument: `args`

(ARG005)